src/components/_FunctionButtons.tsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 
 import InputButton from "./InputButton";
 import {MdFunctions, MdInsertChart} from "react-icons/md";
-import {mdiFunctionVariant} from "@mdi/js";
+import { TbMathFunction } from "react-icons/tb";
```

Comment:
should the formatting be consistent and CI-enforced here?

Reply:
<img width="833" height="452" alt="image" src="https://github.com/user-attachments/assets/564e63bf-c924-475b-9726-9dd4e8776b14" />

Reply:
Cooking something in https://github.com/maplibre/maputnik/pull/1376